Portal/src/Datahub.Functions/AzureConfig.cs

```diff
@@ -83,5 +83,7 @@ public class AdoConfig
     public string PatSecretName { get; set; } = "ado-service-user-pat";
     public string OrgName { get; set; } = "DataSolutionsDonnees";
     public string ProjectName { get; set; } = "FSDH SSC";
-    public string URL { get; set; } = "https://dev.azure.com/{organization}/{project}/_apis/wit/workitems/${workItemTypeName}?api-version=6.0";
+    public string WorkItemsUrlTemplate { get; set; } = "https://dev.azure.com/{organization}/{project}/_apis/wit/workitems/${workItemTypeName}?api-version=6.0";
```

Comment:
Is the `$` in `${workItemTypeName} part of the url? I was going to edit and remove it directly but then I thought it might be some requirement in the ADO url?

Reply:
[Its required in the url](https://learn.microsoft.com/en-us/rest/api/azure/devops/wit/work-items/create?view=azure-devops-rest-7.1&tabs=HTTP)